src/settings/TechnicalForm.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  Form,
+} from 'react-final-form';
+
+import {
+  Pane,
+  PaneFooter,
+  TextField,
+  Checkbox,
+  Button,
+} from '@folio/stripes/components';
+
+import RowComponent from './RowComponent';
+
+class TechnicalForm extends Component {
+  static propTypes = {
+    label: PropTypes.node.isRequired,
+  };
+
+  renderFooter = () => {
+    return (
+      <PaneFooter
+        renderEnd={(
+          <Button
+            data-test-technical-form-button-save
+            type="submit"
+            buttonStyle="primary paneHeaderNewButton"
+            marginBottom0
+          >
+            <FormattedMessage id="stripes-core.button.save" />
+          </Button>
+        )}
+      />
+    );
+  };
+
+  render() {
+    const {
+      label,
+    } = this.props;
+
+    return (
+      <Pane
+        defaultWidth="50%"
+        fluidContentWidth
+        paneTitle={label}
+        footer={this.renderFooter()}
+      >
+        <Form
+          onSubmit={() => {
```

Comment:
I would have added `noop` from lodash for clarity. Check other cases like this.

```suggestion
          }}
```
```suggestion
          onSubmit={noop}
```

Reply:
onSubmit event will be added in scope of https://issues.folio.org/browse/MODOAIPMH-110